subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/repositories_cheat_sheet.adoc

```diff
@@ -0,0 +1,425 @@
+[[repositories-cheat-sheet]]
+= Repositories Cheat Sheet
+
+[TIP]
+====
+Just getting started with repositories?
+Check out the <<declaring_repositories.adoc#declaring-repositories, Declaring Repositories>> to learn the basics of repositories.
+====
+
+[[sec:handling_credentials]]
+== Credentials
+
+Never store credentials in plain text in your build script.
+Instead, store the credentials in a local `gradle.properties` file and use the link:https://plugins.gradle.org/plugin/nu.studer.credentials[credentials plugin] to encrypt and consume credentials.
+
+You can use link:{javadocPath}/org/gradle/api/artifacts/repositories/AuthenticationSupported.html#credentials-java.lang.Class-[AuthenticationSupported.credentials]
```

Comment:
nitpick:
```suggestion
You can use the link:{javadocPath}/org/gradle/api/artifacts/repositories/AuthenticationSupported.html#credentials-java.lang.Class-[AuthenticationSupported.credentials]
```

Reply:
Most documentation omits articles when referring to methods; I'm comfortable leaving this the way it is.